Task2.2/Matrix.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2._2
+{
+    public class Matrix
+    {
+        int[] _elements;
+        public int Size { get; }
+
+        public Matrix(params int[] elements)
+        {
+            if (elements == null)
+            {
+                Size = 0;
+                _elements = [];
+            }
+            else
+            {
```

Comment:
better to allocate memory and copy elements from the original array to _elements